src/components/Attachment/__tests__/Attachment.test.js

```diff
@@ -15,7 +15,7 @@ import {
   generateGiphyAttachment,
   generateImageAttachment,
   generateImgurAttachment,
-} from 'mock-builders/generator/attachment';
+} from '../../../mock-builders/generator/attachment';
```

Comment:
Any reason why are we doing this?

Reply:
for consistency with the way we import local files everywhere else